app/src/main/java/com/google/android/soundchecker/dsd/DsfReader.kt

```diff
@@ -147,9 +148,15 @@ class DsfReader(inputStream: InputStream) {
     private fun readNextBlock(channel: Int): Boolean {
         Arrays.fill(mData[channel], 0, mData[channel].size - 1, 0x0.toByte())
         try {
+            if (mDataLeft < mData[channel].size && mStream.markSupported()) {
+                Log.d(TAG, "Reach end of file, mark supported, reset to beginning")
```

Comment:
Add the word loop somewhere as it's not immediately obvious that the audio will continue to play after resetting to the beginning.